benefits/core/migrations/0002_sample_data.py

```diff
@@ -1,8 +1,17 @@
+"""Data migration which loads sample data.
+Set environment variable DJANGO_LOAD_SAMPLE_DATA to False to skip loading sample data.
+"""
+import os
+
 from django.db import migrations
 from django.utils.translation import gettext_lazy as _
 
 
-def load_initial_data(app, *args, **kwargs):
+def load_sample_data(app, *args, **kwargs):
+    if os.environ.get("DJANGO_LOAD_SAMPLE_DATA", "true").lower() == "false":
```

Comment:
This should probably be wrapped in a regular setting like we have done for other environment variables.